src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -119,6 +119,7 @@ public FeatureOnOffOptionsProvider()
             FeatureOnOffOptions.UseEnhancedColors,
             FeatureOnOffOptions.AddImportsOnPaste,
             FeatureOnOffOptions.OfferRemoveUnusedReferences,
+            FeatureOnOffOptions.AutomaticallyCompleteStatementOnSemicolon,
```

Comment:
can you extract this into a separate PR?

Reply:
removed it